samples/file-encryptor/CMakeLists.txt

```diff
@@ -26,10 +26,10 @@ add_custom_target(sign ALL DEPENDS enclave/enclave.signed)
 
 if ((NOT DEFINED ENV{OE_SIMULATION}) OR (NOT $ENV{OE_SIMULATION}))
   add_custom_target(run
-    DEPENDS file-encryptor_host sign
+    DEPENDS file-encryptor_host sign ${CMAKE_CURRENT_SOURCE_DIR}/testfile
```

Comment:
I don't think we need `${CMAKE_CURRENT_SOURCE_DIR}` here...?

Reply:
Nope. Thanks for catching it!